schematic/store/synapse.py

```diff
@@ -2705,9 +2706,6 @@ def replaceTable(
                 current_table.addColumn(col)
             self.synStore.syn.store(current_table, isRestricted=self.restrict)
 
-            # wait for synapse store to finish
-            sleep(1)
```

Comment:
.store is a blocking call so I'm not sure why this was added here. 

Do any FAIR folks have context on if this is actually needed?

Reply:
This was added here: https://github.com/Sage-Bionetworks/schematic/commit/729c4acce9dfafddbd6c648cac177a7388664d9b.  Tables and Fileviews can be tricky to work especially when storing to large fileviews/ tables - that said, it _might be_ ok to remove.

Not sure if this work was attached to a github issue or jira ticket.

Reply:
yeah this and the other sleep were added because we ran into issues when we tried to pull views too soon after they were modified or the projects were modified. I'll let @BryanFauble decide if they're still needed because he's seeing the performance but before the sleeps were added it was really inconsistent on whether or not the view would be ready and up to date

Reply:
Thanks @GiaJordan for the explanation! I'm inclined to keep this here unless we test this on deployment, but we could add comments here about why this is here.

Reply:
I will leave these in for now then